FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -230,31 +247,53 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                        </msItem> 
-                            <layoutDesc></layoutDesc>
+                        </msContents>
+                        <physDesc>
+                            <objectDesc form="Codex">
+                                <supportDesc>
+                                    <support>
+                                        <material key="parchment"/>
+                                    </support>
+                                </supportDesc>
+                            
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="21" corresp="#p1_i1"></layout>
+                                
+                                <layout columns="2" writtenLines="35" corresp="#p1_i2"></layout>
+                                <layout columns="2" writtenLines="30" corresp="#p1_i3"></layout>
+                                
+                            </layoutDesc>
+                            </objectDesc>
                             <handDesc><handNote xml:id="h1" script="Ethiopic">
                                 <ab type="script"/>
                                 <date when="1728">Dated according to a colophon on folio <locus target="#122r"/></date>
                                 <locus from="2r" to="2v"/> and <locus from="7r" to="122rb"/> and <locus from="123ra" to="128vb"/>
                                 <note>Contrary to <bibl><ptr target="bm:Marrassini1986Laurenziana"/><citedRange unit="page">93</citedRange></bibl>, who considered textual unit 1 (Miracles of Mary) on 
                                     <locus from="2" to="113"/> of different hand as textual unit 2 (Miracles of ʾEwosṭātewos) on <locus from="113vb" to="122v"/> and other notes on <locus from="123ra" to="128v"/></note>
                             </handNote>     </handDesc>
-                            <objectDesc></objectDesc>
-                            <addition></addition>
-                            <history></history>
-                        <msPart xml:id="p2"><msIdentifier><msContents><summary><msItem xml:id="p2_i1">
+                            </physDesc>
+                            
+                            </msPart>
+                        <msPart xml:id="p2"><msIdentifier/><msContents><summary/><msItem xml:id="p2_i1">
                         <locus from="3ra" to="6va"></locus>
-                            <layout columns="2" writtenLines="25"></layout>
-                            <title type="incomplete" ref="LIT2384Taamme">Miracles of Mary<title/>
+                            
+                            <title type="incomplete" ref="LIT2384Taamme">Miracles of Mary</title>
                                 <incipit xml:lang="gez">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩፡ አምላክ።
-                                    እምቅድመ፡ አምር፡ ይትነበብ፡ መቅድመ፡ ተአምሪሃ፡ ለእግዝእትነ፡ ማርያም፡ ዘበዕብራይስጢ፡ ማሪሃም፡ ዝንቱ፡ ሥርዓት፡ ዘወጽአ፡ እመካነ፡ ማልቃ፡ ዘምስር።</incipitxm>
```

Comment:
try to pay attention that the closing tag spells the same as the opening one. Oxygen should help you by underlining what is wrong